subprojects/docs/src/docs/stylesheets/userGuideHtml.xsl

```diff
@@ -174,6 +174,7 @@
                                 <ul id="managing-dependencies">
                                     <li><a href="/userguide/introduction_dependency_management.html">Introduction to Dependency Management</a></li>
                                     <li><a href="/userguide/declaring_dependencies.html">Declaring Dependencies</a></li>
+                                    <li><a href="/userguide/defining_using_configurations.html">Defining and Using Configurations</a></li>
```

Comment:
Optional: perhaps "Managing Dependency Configurations" would clarify the type of configurations we're talking about.

Reply:
Also this link does not work, it should be defining_and_using_configurations.html

Reply:
> Optional: perhaps "Managing Dependency Configurations" would clarify the type of configurations we're talking about.

Maybe that makes it more clear that we are talking about configurations in the context of dependency management. done

> Also this link does not work, it should be defining_and_using_configurations.html

done